spec/unit/type/user_spec.rb

```diff
@@ -351,13 +351,20 @@ def self.instances; []; end
       @user = described_class.new(:name => 'foo', :comment => @value)
     end
 
-    it "should be converted to ASCII_8BIT for ruby 1.9 / 2.0", :if => RUBY_VERSION < "2.1.0" && String.method_defined?(:encode) do
-      expect(@user[:comment].encoding).to eq(Encoding::ASCII_8BIT)
-      expect(@user[:comment]).to eq(@value.force_encoding(Encoding::ASCII_8BIT))
-    end
-
-    it "must not be converted for ruby >= 2.1", :if => RUBY_VERSION >= "2.1.0" do
-      expect(@user[:comment].encoding).to eq(Encoding::UTF_8)
+    it "should define the #insync?(is,should) method" do
```

Comment:
TBH I'm kind of skeptical of these unit tests and think I'll be adding acceptance tests to really validate the behavior here.